.github/workflows/ci-it.yml

```diff
@@ -41,4 +41,4 @@ jobs:
       - name: Run Test
         run: |
           go test -work -cpu=2 -timeout 3s -race -coverprofile=coverage.txt -covermode=atomic ./...
-          bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+          bash <(curl -s https://codecov.io/bash)
```

Comment:
Each PR only needs to do one thing. 
Others have fixed the problem of lint failure, so you don't need to change ci-it.yml